tests/nlu/test_components.py

```diff
@@ -190,6 +191,7 @@ def test_can_handle_language_guard_clause(
     assert expected_exec_msg in str(excinfo.value)
 
 
+@pytest.mark.trains_model
```

Comment:
mhm, I don't think it makes sense to lable this one.

Reply:
Our marking strategy has to be consistent if we want to automate it. And we agreed on any call to `nlu.train` or `code.train`.

Reply:
Ok 👍🏻